.travis.yml

```diff
@@ -1,6 +1,12 @@
 before_install:
-    - gem install bundler
+  - gem install bundler
 language: ruby
 rvm:
-    - 2.3.0
-script: bundle exec rspec spec
\ No newline at end of file
+  - 1.9.3
+  - 2.0
+  - 2.1
```

Comment:
Support for Ruby 2.1 has ended in [April 2017](https://www.ruby-lang.org/en/news/2017/04/01/support-of-ruby-2-1-has-ended/) and we do not intend to support it either.